CHANGELOG.md

```diff
@@ -1,9 +1,9 @@
 # Upcoming
 
 * BREAKING CHANGE: Seperate the code from the angular_token_auth into seperate files.
-* Add file linting.
```

Comment:
This is still a valid line in the changelog. It can be expanded with "using eslint"